dotCMS/src/main/java/com/dotcms/contenttype/business/UniqueFieldAPI.java

```diff
@@ -0,0 +1,21 @@
+package com.dotcms.contenttype.business;
+
+import com.dotmarketing.exception.DotDataException;
+
+/**
+ * This API allow you to interact with the unique_fields table
+ */
+public interface UniqueFieldAPI {
```

Comment:
Do we really need a separate API for unique fields? Why not using the existing Field API?

Reply:
I think it is ok but should be called UniqueAPI and be able to use with other things not only field
at the end it is key value stuff

Reply:
done https://github.com/dotCMS/core/pull/30332/commits/504efaaeb01448b4d59b753e792b519af0312212